S/SCIP_PaPILO/build_tarballs.jl

```diff
@@ -4,25 +4,23 @@ using BinaryBuilder, Pkg
 
 name = "SCIP_PaPILO"
 
-version = v"800.100.000"
+version = v"900.000.000"
 
+# Collection of sources required to complete build
 sources = [
     ArchiveSource(
-        "https://scipopt.org/download/release/scipoptsuite-8.1.0.tgz",
-        "a3c1b45220252865d4cedf41d6327b6023608feb360d463f2e68ec4ac41cda06"
+        "https://scipopt.org/download/release/scipoptsuite-9.0.0.tgz",
+        "c49a0575003322fcbfe2d3765de7e3e60ff7c08d1e8b17d35409be40476cb98a"
     ),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd scipoptsuite*
 
-# for soplex threadlocal
-export CXXFLAGS="-DTHREADLOCAL=''"
-
-# can be removed for scip 805
-echo "target_link_libraries(clusol gfortran)" >> papilo/CMakeLists.txt
-
+# for soplex threadlocal and large files
+export CXXFLAGS="-DTHREADLOCAL='' -Wa,-mbig-obj"
```

Comment:
I think you need to make adding the `-Wa,-mbig-obj` only added on the mingw builds.